apps/vessel-history/features/profile/components/ForcedLabor.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo } from 'react'
+import cx from 'classnames'
+import { Trans, useTranslation } from 'react-i18next'
+import { RiskLevel, RiskOutput, VesselWithHistory } from 'types'
+import { VesselFieldLabel } from 'types/vessel'
+import DataAndTerminology from 'features/data-and-terminology/DataAndTerminology'
+import styles from './Info.module.css'
```

Comment:
This component should have its own css module file since it does not share/reuse any styles from the`Info.module.css`